From 8fda670234bcc2fd5450f790290aad64115f16de Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Thu, 1 Aug 2013 13:15:56 +0800
Subject: [PATCH 226/602] ENGR00273437 add LegacyCamera into PRODUCT_PACKAGES

add LegacyCamera into PRODUCT_PACKAGES, we need
the LegacyCamera on no 3D chip such as mx6sl.

Signed-off-by: Richard Liu <r66033@freescale.com>
---
 imx6/imx6.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/imx6/imx6.mk b/imx6/imx6.mk
index 0cd2c18..d735818 100644
--- a/imx6/imx6.mk
+++ b/imx6/imx6.mk
@@ -14,6 +14,7 @@ PRODUCT_PACKAGES += \
 	Gallery		    			\
 	SoundRecorder				\
 	Camera					\
+        LegacyCamera                            \
 	Email					\
 	FSLOta					\
 	CactusPlayer                            \
-- 
1.8.0

